internal/services/network/subnet_data_source.go

```diff
@@ -83,6 +84,11 @@ func dataSourceSubnet() *pluginsdk.Resource {
 	}
 
 	if !features.FourPointOhBeta() {
+		resource.Schema["private_endpoint_network_policies_enabled"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		}
```

Comment:
we should add a deprecation message here?

Reply:
Added